.env.example

```
@@ -1,17 +1,16 @@
 NODE_ENV=dev|prod
 TOKEN
 MONGO_URL
-TIME_API
 PAYME_URL
 PAYME_ENDPOINT
 CARD
 DISCORD_WEBHOOK_URL=https://discord.com/api/webhooks/id/token
+DISCORD_FEEDBACK_THREAD_ID=XXXX
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_FEEDBACK_THREAD_ID key should go before the DISCORD_WEBHOOK_URL key

(UnorderedKey)